java/web/src/main/resources/applicationContext.xml

```diff
@@ -30,7 +30,7 @@
     </bean>
 
     <bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
-        <property name="order" value="10"/>
+        <property name="order" value="0"/>
```

Comment:
curiosity: there was some sort of conflict?

Reply:
I noticed ldap properties was not read in c074 and when people customzed the binary to use LDAP, so I bring it back to 0. 
I don't remember why I set it to 10.